webhook/pkg/mutating_webhook/remove-resourcespec.go

```diff
@@ -0,0 +1,59 @@
+package mutating_webhook
+
+import (
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	DEFAULT_RUNTIME_CLASS_NAME  = "kata-remote-cc"
+	VM_ANNOTATION_INSTANCE_TYPE = "kata.peerpods.io/instance_type"
+	VM_INSTANCE_TYPE_DEFAULT    = "t2.small"
+	VM_EXTENDED_RESOURCE        = "kata.peerpods.io/vm"
```

Comment:
Not a big problem for now, but I wonder if we should move away from kata to cc/confidential-containers?

Reply:
Yeah, this is a good suggestion. I'll make the changes

Reply:
@stevenhorsman re-visiting the extended resource naming aspect, I'm thinking of leaving it as-is for now and get some more ideas before finalising the name :-)